.github/workflows/build.yml

```diff
@@ -0,0 +1,20 @@
+name: ci
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - run: pip install mkdocs
+      - run: pip install mkdocs-material
+      - run: pip install pymdown-extensions
+      - run: pip install MarkdownHighlight
+      - run: pip install mkdocs-redirects
```

Comment:
Should the python dependencies be installed from `requirements.txt`? I notice there are more in there then here, can we trim the file and use it?

Reply:
Done